styles/_body.scss

```diff
@@ -0,0 +1,47 @@
+.Description {
+    font-family: "Akaya Telivigala";
+    font-weight: 100;
+    text-decoration-line:underline;
+    font-size: 24px;
+    text-shadow: 1px 0 0 #000, 0-1px 0 #000, -1px 0 0 #000;
+    color: $SecondaryColor;
+    grid-area: Description;
+    justify-content: center;
```

Comment:
`justify-content` or `align-items` works only in flexbox. You have to use CSSGrid now